tests/foreman/ui/test_org.py

```diff
@@ -222,14 +248,15 @@ def test_positive_delete(self, org_name):
         @assert: Organization is deleted successfully
 
         """
-        entities.Organization(name=org_name).create()
         with Session(self.browser) as session:
-            session.nav.go_to_org()
-            self.org.remove(org_name)
-
-    @skip_if_bug_open('bugzilla', 1225588)
-    @data(*generate_strings_list())
-    def test_positive_delete_bz1225588(self, org_name):
+            for org_name in generate_strings_list():
+                with self.subTest(org_name):
+                    # Use nailgun to create org
+                    entities.Organization(name=org_name).create()
+                    session.nav.go_to_org()
+                    self.org.remove(org_name)
```

Comment:
Not required for this PR but it would be nice if we performed a search and an assertion after the organization gets deleted